data/interfaces.txt

```diff
@@ -429,6 +430,14 @@ interface "preferences"
 		dimensions 0 -200
 		color "energy"
 		size 3
+	visible if "show plugins changed"
+	sprite "ui/planet dialog button"
+		center 0 210
+	visible if "show plugins changed"
```

Comment:
```suggestion
```

The visible if condition stays until you change it, so this one is unnecessary.

Reply:
I'll want to test